app/docs/0.12.x/admin-api.md

```diff
@@ -2100,7 +2173,7 @@ target](#delete-target) instead.
 
 #### Endpoint
 
-<div class="endpoint post">/upstreams/{upstream name or id}/targets/{target or id}/healthy</div>
+<div class="endpoint post">/upstreams/{upstream name or id}/targets/{target or id}/unhealthy</div>
```

Comment:
oops :/ I think this also belongs to master in the 0.12 version of the docs? So we can deploy it asap? (I know you opened the other PR against master originally)

Reply:
PR to master: https://github.com/Kong/getkong.org/pull/604